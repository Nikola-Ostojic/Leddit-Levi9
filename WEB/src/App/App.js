import React from "react";
import { Provider } from "react-redux";

import { ToastContainer } from "react-toastify";

import AppNavigation from "navigation/AppNavigation";
import "react-toastify/dist/ReactToastify.css";
import store from "store";

export const App = () => (
  <Provider store={store}>
    <AppNavigation />
    <ToastContainer autoClose={2500} />
  </Provider>
);

export default App;
