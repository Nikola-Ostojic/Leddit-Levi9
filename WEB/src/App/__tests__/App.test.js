import { shallow } from "enzyme";
import React from "react";

import App from "../App";

describe("App tests", () => {
  it("Should match snapshot", () => {
    const wrapper = shallow(<App />);

    expect(wrapper).toMatchSnapshot();
  });
});
