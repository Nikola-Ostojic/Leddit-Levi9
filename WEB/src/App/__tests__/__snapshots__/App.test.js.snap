// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App tests Should match snapshot 1`] = `
<Provider
  store={
    Object {
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
      Symbol(observable): [Function],
    }
  }
>
  <Connect(AppNavigation) />
  <ToastContainer
    autoClose={2500}
    closeButton={[Function]}
    closeOnClick={true}
    draggable={true}
    draggablePercent={80}
    hideProgressBar={false}
    newestOnTop={false}
    pauseOnFocusLoss={true}
    pauseOnHover={true}
    position="top-right"
    role="alert"
    rtl={false}
    transition={[Function]}
  />
</Provider>
`;
